docs/metrics/getting-started-prometheus-grafana/README.md

```diff
@@ -71,7 +98,9 @@ end
 
 Instrument --> | Measurements | MeterProvider
 
-MeterProvider --> | Metrics | MetricReader --> | Pull | PrometheusHttpListener
+MeterProvider --> | Metrics | MetricReader --> | Push | OtlpExporter
```

Comment:
Talking about MetricReader feels like a level of detail unneeded for the tutorial. I expect user's mental model is something like:
Recorded measurements -> OpenTelemetry -> OtlpExporter -> Remote Prometheus server

Reply:
See if this is better https://github.com/open-telemetry/opentelemetry-dotnet/pull/4878.